Source/RapyutaSimulationPlugins/Private/Sensors/RR2DLidarComponent.cpp

```diff
@@ -6,11 +6,13 @@
 #include "Kismet/KismetMathLibrary.h"
 #include "PhysicalMaterials/PhysicalMaterial.h"
 
+#include "Tools/RRROS2LidarPublisher.h"
+
 #include <limits>

```

Comment:
// UE, //rclUE, //RapyutaSimulationPlugins for avoiding auto clang-format reordering?

Reply:
// UE, //std, //RapyutaSimulationPlugins for avoiding auto clang-format reordering?